class-tgm-plugin-activation.php

```diff
@@ -1337,6 +1362,22 @@ protected function display_settings_errors() {
 		}
 
 		/**
+		 * Display admin notice if plugins have been force activated.
+		 *
+		 * @since 2.6.0
+		 */
+		public function display_forced_activation_notice() {
+			if ( $force_activated_plugins = get_transient( 'tgmpa_force_activated_plugins' ) ) {
```

Comment:
Please don't assign to a variable within a condition. I know the WP project does this regularly, but it's bad practice.